plugins/hm-stack/inc/rest-controller/namespace.php

```diff
@@ -160,19 +160,29 @@ function get_environment_data() {
 	if ( is_wp_error( $stack_data ) ) {
 		// Prevent constant re-fetching in the event of a failure.
 		wp_cache_set( 'activity', $stack_data, 'hm-stack', 5 * \MINUTE_IN_SECONDS );
-		return $this->get_wp_error_for_hm_stack_return( $stack_data );
+		return get_wp_error_for_hm_stack_return( $stack_data );
 	}
 
+	global $wpdb;
+
 	$data = [
 		'environment_data' => [
+			'wordpress'     => $stack_data['wordpress-version'],
+			'hmplatform'    => $stack_data['hm-platform-version'],
+			'architecture'  => $stack_data['architecture'],
+			'version'       => $stack_data['version'],
 			'elasticsearch' => '', // Awaiting this availability
 			'php'           => substr( phpversion(), 0, 5 ),
-			'mysql'         => '', // Will add this later.
+			'mysql'         => $wpdb->db_version(),
```

Comment:
This is meant to be the version of mysql, I think `db_version` is the WordPress database version, maybe?

Reply:
No, I believe `$wpdb->db_version()` gives the MySQL version.